modules/azure_aks/variables.tf

```diff
@@ -1,81 +1,123 @@
-variable aks_cluster_name {}
+variable "aks_cluster_name" {
+  description = "The name for the AKS resources created in the specified Azure Resource Group"
+  type        = string
+}
+
+variable "aks_cluster_rg" {
+  description = "The resource group name to be imported"
+  type        = string
+}
 
-variable aks_cluster_rg {}
-variable aks_cluster_rg_id {}
-variable aks_cluster_dns_prefix {}
+variable "aks_cluster_rg_id" {
+  description = "The `azurerm_kubernetes_cluster`'s id."
+  type        = string
+}
+
+variable "aks_cluster_dns_prefix" {
+  description = "DNS prefix specified when creating the managed cluster."
+  type        = string
+}
 
 variable "aks_cluster_location" {
   description = "The Azure Region in which all resources in this example should be provisioned"
+  type        = string
   default     = "eastus"
 }
 
-variable aks_cluster_sku_tier {
+variable "aks_cluster_sku_tier" {
   description = "The SKU Tier that should be used for this Kubernetes Cluster. Possible values are Free and Paid (which includes the Uptime SLA). Defaults to Free"
+  type        = string
   default     = "Free"
 
   validation {
-    condition     = contains(["Free", "Paid"],  var.aks_cluster_sku_tier)
+    condition     = contains(["Free", "Paid"], var.aks_cluster_sku_tier)
     error_message = "ERROR: Valid types are \"Free\" and \"Paid\"!"
   }
 }
 
 variable "aks_private_cluster" {
-  default = false
+  description = "The cluster API endpoint uses Private IP address?"
```

Comment:
description seems unclear and ends with a question mark

Reply:
Updated the description in the latest commit.